en/docs/observe/mi-observe/working-with-management-api.md

```diff
@@ -1519,6 +1810,34 @@ The management API has multiple resources to provide information regarding the d
 	</endpoint>
   	```
 
+	??? note "Search for a Registry Resource by name"
+		You can also search for a Registry Resource by name. To do that, use the following example cURL command.
+
+		```bash tab='Request'
+		curl -X GET "https://localhost:9164/management/registry-resources?path=registry/config&searchKey=test-text" -H "accept: application/json" -H "Authorization: Bearer $AccessToken" -k -i
+
+		```
+
+		```base tab='Response'
+		{
+			"list": {
+				"name": "config",
+				"files": [{
+					"name": "testFolder",
+					"files": [{
+						"name": "test-text.txt",
+						"files": [],
+						"type": "testMediaType"
+					}],
+					"type": "directory"
+				}],
+				"type": "directory"
+			}
+		}
+
+		```
+
+
```

Comment:
Also we need to mention the search capabilities to the carbon applications, logs, users and roles as well, pointing out the corresponding curl commands

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/6433/commits/5f196449167f0d69798c32ca17b2b61470553d0b